### PR TITLE
Code for evaluation functions

### PR DESCRIPTION
**Pull request by LeoGaller**
Code for assesment on quantile loss and proportion of hits.

### CODE REVIEW

File: tests/evaluation_galler.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Structure for quantile loss evaluator
+def quantile_loss_evaluator( df , predict_col , target_col , tau ):
+    
+    # assigning the values
+    y_hat = df[predict_col]
+    y_tru = df[target_col]
+    
+    # Calculating
+    quantile_loss = { str(tau) : sum((tau - (y_tru - y_hat )) * (y_hat)) }
```
Comment (matheusfacure) [inline tests/evaluation_galler.py]: Melhor usar a média pra loss n ser sensível ao tamanho da amostra

Comment (matheusfacure) [reply tests/evaluation_galler.py]: Also, sua fórmula tá errada. O certo é `(tau-(y_true < y_hat)) * (y_true-y_hat))`

Comment (matheusfacure) [reply tests/evaluation_galler.py]: De novo, sua função tá no lugar errado. Tá em testes e tem que tá no src da lib.
